polysemy-RandomFu/src/Polysemy/RandomFu.hs

```diff
@@ -89,11 +91,11 @@ runRandomIO = interpret $ \case
 ------------------------------------------------------------------------------
 -- | Run in 'IO', using the given 'R.PureMT' source, stored in an 'IORef'
 runRandomIOPureMT
-  :: MonadIO (Sem r)
+  :: Member (Lift IO) r
```

Comment:
Why these changes?  I think the two versions are equivalent?

Reply:
Because `runRandomIOPureMT` uses `runRandomSource`, which requires this constraint. Can't compile without this change.

Reply:
Right!  I hadn't looked carefully enough.